.github/workflows/formatting.yml

```diff
@@ -19,3 +19,19 @@ jobs:
         python -m pip install pre-commit
     - name: Run pre-commit
       run: python -m pre_commit run --all-files
+  formatting-check:
```

Comment:
do we need this? aren't these formatting checks already exectude with the pre-commit hook?